### PR TITLE
New Readme and Contributing

### LINKED ISSUE #9: README

### PR DESCRIPTION
**Pull request by Bouncey**
New Readme and Contributing

closes #9

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# CatPhotoApp Contributing
+
+### Fork the repo
```
Comment (ghost) [inline CONTRIBUTING.md]: Instead of using `###`, use a numbering system e.g. `1. Fork the repo`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+### Clone your fork
+
+###  Branch off master
```
Comment (ghost) [inline CONTRIBUTING.md]: Instead of using `###`, use a numbering system e.g. `1. Fork the repo`.

Add a git snippet here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# CatPhotoApp Contributing
+
+### Fork the repo
+
+### Clone your fork
```
Comment (ghost) [inline CONTRIBUTING.md]: Instead of using `###`, use a numbering system e.g. `1. Fork the repo`.

Add a git snippet here.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+###  Branch off master
+
+#### `npm install`
+
+#### `gulp`
```
Comment (ghost) [inline CONTRIBUTING.md]: Instead of using `###`, use a numbering system e.g. `1. Fork the repo`.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# CatPhotoApp Contributing
+
+### Fork the repo
+
+### Clone your fork
+
+###  Branch off master
+
+#### `npm install`
```
Comment (ghost) [inline CONTRIBUTING.md]: Instead of using `###`, use a numbering system e.g. `1. Fork the repo`.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+#### `gulp`
+
+This will get you running on `localhost:8080`
+
+[Squash](https://www.freecodecamp.com/wiki/en/git-squash/) your commits into one.
```
Comment (ghost) [inline CONTRIBUTING.md]: You don't really need to add this, most contributors for this project will know git. If we starting adding git details, we need to expand etc.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+## Thanks for contributing :tada:
```
Comment (ghost) [inline CONTRIBUTING.md]: You could remove the heading 2 at the start :wink:

File: README.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+# CatPhotoApp
+
+This is the development repo for @FreeCodeCamp 's CatPhotoApp
```
Comment (ghost) [inline README.md]: ~~This is the development repo for @FreeCodeCamp 's CatPhotoApp~~ This is the development repo for [FreeCodeCamp](https://freecodecamp.com)'s CatPhotoApp

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+# CatPhotoApp Contributing
```
Comment (ghost) [inline CONTRIBUTING.md]: ~~CatPhotoApp Contributing~~ Contributing Guide

Comment (Bouncey) [inline CONTRIBUTING.md]: Oops, forgot that one

File: README.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+# CatPhotoApp
+
+This is the development repo for [FreeCodeCamp](freecodecamp.com)'s CatPhotoApp.
```
Comment (ghost) [inline README.md]: ~~`[FreeCodeCamp](freecodecamp.com)'s`~~ `[FreeCodeCamp](https://freecodecamp.com)'s`

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+3.  Branch off master
+
+```
+$ git checkout master
+
+$ git checkout -b <-your branch name->
+
+```
+
+4. `$ npm install`
```
Comment (ghost) [inline CONTRIBUTING.md]: ~~`4. $ npm install`~~ `4. npm install`

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+
+$ cd catphotoapp
+
+```
+
+3.  Branch off master
+
+```
+$ git checkout master
+
```
Comment (ghost) [inline CONTRIBUTING.md]:
```
 $ git checkout master

 $ git checkout -b <-your branch name->

```

``` sh
git checkout master

git checkout -b <-your branch name->
```

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+```
+
+3.  Branch off master
+
+```
+git checkout master
+
+git checkout -b <-your branch name->
+
```
Comment (ghost) [inline CONTRIBUTING.md]: Remove this line

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
+# Contributing Guide
+
+1. Fork the repo
+
+2. Clone your fork
+
+```
```
Comment (ghost) [inline CONTRIBUTING.md]: Tab all of this code snippet.
The numbering is messed up.

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+git clone <-your fork URL-> catphotoapp
+
+cd catphotoapp
+
+```
+
+3.  Branch off master
+
+```
+git checkout master
```
Comment (ghost) [inline CONTRIBUTING.md]: Tab all of this code snippet.
The numbering is messed up.

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (ghost) [inline CONTRIBUTING.md:16]: Remove this line

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -0,0 +1,30 @@
+# Contributing Guide
+
+1. Fork the repo
+
```
Comment (ghost) [inline CONTRIBUTING.md]: Remove this line

File: CONTRIBUTING.md [hunk 18]
```diff
@@ -0,0 +1,30 @@
...
+```
+git checkout master
+
+git checkout -b <-your branch name->
+
+```
+
+4. `npm install`
+
```
Comment (ghost) [inline CONTRIBUTING.md]: Remove this line.

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks for contributing :tada:

**Pull request merged**